src/arch/xtensa/boot_loader.c

```diff
@@ -78,6 +80,11 @@ static void parse_module(struct sof_man_fw_header *hdr,
 			bbzero((void *)mod->segment[i].v_base_addr,
 			       mod->segment[i].flags.r.length *
 			       HOST_PAGE_SIZE);
+			/* save fw end address to be used in memory pm */
+			fw_size_in_use = mod->segment[i].v_base_addr
```

Comment:
Please do not pass values via static variables

Reply:
Also it can break if in some point in the future some1 is going to parse more modules or in different order.

Reply:
BTW why do we take just bss size? I don't remember how memory layout looks, but shouldn't we take sum of bss+text?

Reply:
BSS is las segment in memory